docs/topics/native/native-memory-manager.md

```diff
@@ -32,10 +37,10 @@ and waits for its completion.
 ### Monitor GC performance
 
 To monitor the GC performance, you can look through its logs and diagnose issues. To enable logging,
-set the following compiler option in the Gradle build script:
+set the following compiler option in your `gradle.properties` file:
 
 ```none
--Xruntime-logs=gc=info
+kotlin.native.runtimeLogs.gc=info
```

Comment:
Unfortunately, it doesn't work this way, this is a different family of options. Unlike the `binary` ones, these have to be passed through the command line (e.g. in the `freeCompilerArgs`) in the form of `-Xruntime=logs=gc=info`.